Makefile

```diff
@@ -160,16 +160,6 @@ else
 	cd standalone && $(NPM) run debug;
 endif
 
-## Builds the webapp on ci -- dependencies are handled by the npm-dependencies step in ci
-.PHONY: webapp-ci
-webapp-ci:
-	cd webapp && $(NPM) run build
-
-## Builds the standalone apps on ci -- dependencies are handled by the npm-dependencies step in ci
-.PHONY: standalone-ci
-standalone-ci:
-	cd standalone && $(NPM) run build
-
```

Comment:
Was this intentional? This means we'll be rerunning the webapp/node_modules for ci

Reply:
I assumed that would be cached and wouldn't really matter. It was actually failing as it was probably because the way we do caching is slightly changed now.

Reply:
Hm, I guess I'm more familiar with the circleci way (where we would pull in hashed caches). But if this works, then I guess the extra work isn't too bad (?).